numba_dppy/tests/test_dpnp_functions.py

```diff
@@ -1411,5 +1411,84 @@ def f(a, b):
         self.assertTrue(max_abs_err < 1e-4)
 
 
+@unittest.skipUnless(
+    ensure_dpnp() and dpctl.has_gpu_queues(), "test only when dpNP and GPU is available"
+)
+class Testdpnp_array_ops_functions(unittest.TestCase):
+    tys = [np.int32, np.uint32, np.int64, np.uint64, np.float, np.double]
+
+    def check_take_for_different_datatypes(
+        self, fn, test_fn, ind, dims, tys, matrix=None
+    ):
+        for ty in tys:
+            if matrix:
+                a = np.arange(np.prod(np.array(dims)), dtype=ty).reshape(
```

Comment:
If `dims` is expected to be a `np.array` may be let us pass an `np.arra`y instead of a `list`.